packages/grid/x-data-grid/src/models/api/gridSortApi.ts

```diff
@@ -53,6 +53,7 @@ export interface GridSortApi {
    * The index is based on the sorted but unfiltered row list.
    * @param {GridRowId} id The `GridRowId` of the row.
    * @returns {number} The index of the row.
+   * @deprecated Use `unstable_getRowIndexRelativeToCurrentPage` instead.
```

Comment:
We advise users to use a private API instead of a public one ?
Does this mean that `unstable_getRowIndexRelativeToCurrentPage` will become public at some point ?

Reply:
It would be public in v6 and replace `getRowIndex`. I made it public now.